bindings/nodejs/native/Cargo.toml

```diff
@@ -1,12 +1,12 @@
-[workspace]
-
 [package]
-name = "iota-client"
+name = "iota-client-nodejs"
```

Comment:
Here it's the same issue as for wallet.rs `The crate name will be read from the npm_package_name environment variable. If the package name includes a namespace (@namespace/package), the namespace will be removed when matching the crate name (package).`

Reply:
So it's `cargo-cp-artifact` having such behavior. iota.rs does not use it during the Node.js / WASM bindings build. Will it be added in the future?

Reply:
I'm fixing this anyways, we don't publish the crates.